examples/demo/pingpong/src/pingpong/j/cpp/main.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+
+#include "Env.hpp"
+#include "deployment.hpp"
+#include "runtime/Action.hpp"
+#include "EventStructures.hpp"
+#include "Game.hpp"
+
+int main()
+{
+  Env::initEnvironment();
+  UsedRuntimePtr rt = UsedRuntimeType::getRuntimeInstance();
+
```

Comment:
Unnecessary newline. This is a demo, we have to make it pretty.

Reply:
I have deleted the next empty line..